src/main/java/com/fakng/fakngagrgtr/parser/apple/VacancyDTO.java

```diff
@@ -0,0 +1,38 @@
+package com.fakng.fakngagrgtr.parser.apple;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@JsonIgnoreProperties({"localeInfo"})
```

Comment:
Do we need this, if we added that ObjectMapper property?

Reply:
No, I fogot☹️